04.bowling/bowling.rb

```diff
@@ -0,0 +1,66 @@
+def main
+  point = get_point(ARGV[0])
+  puts point
+end
+
+def get_point(score_str)
+  shots = get_shots(score_str)
+  frames = get_frames(shots)
+  calc_total_point(frames)
+end
+
+def get_shots(score_str)
+  scores = score_str.split(',')
+  scores.flat_map do |s|
+    s == 'X' ? [10, 0] : s.to_i
+  end
+end
+
+def get_frames(shots)
+  frames = shots.each_slice(2).to_a
+
+  if frames.length > 10
+    final_frame = frames[9..].flat_map do |frame|
+      frame[0] == 10 ? 10 : frame
+    end
+    frames = [*frames[..8], final_frame]
+  end
+  frames
+end
+
+def calc_total_point(frames)
+  point = 0
+  (0..8).each do |i|
```

Comment:
```suggestion
  point = (0..8).sum do |i|
```

のようにsumメソッドを活用するといいかも？

Reply:
なるほど、ご指摘ありがとうございます。修正しました。
https://github.com/mayumonj/ruby-practices/pull/3/commits/1637707c5fc2fa6b46b9225c160787d882e952cb